src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -165,4 +178,13 @@ private Map<EventKey, Event> parseEventRecordsToEventsEntityMap(Set<String> even
         });
     return eventsMap;
   }
+
+  private void ensureOptIn(String orgId) {
+    try {
+      optInController.optInByOrgId(orgId, OptInType.PROMETHEUS);
+    } catch (Exception e) {
+      log.warn("Error while attempting to automatically opt-in orgId={}", orgId);
+      log.debug("Opt-in error for orgId=" + orgId, e);
```

Comment:
Please just consolidate these to one `log.error` message.  If we can't opt-in, that's an error and we want to fix it quickly.

Reply:
Done